lib/pages/home_page.dart

```diff
@@ -110,8 +133,12 @@ class _HomePageState extends State<HomePage> {
       NavigationAction(
         title: 'Etkinlik',
         icon: Icons.celebration,
-        focusNode: focusNodes[0],
-        onPressed: () => _scrollToSection(focusNodes[0]),
+        focusNode: focusNodes[eventPathInformation.focusNodeIndex],
+        onPressed: () {
+          _scrollToSection(focusNodes[eventPathInformation.focusNodeIndex]);
+          widget.pageTitleNotifier.value = eventPathInformation.pathSegmentName;
```

Comment:
the `pageTitleNotifier.value` is only updating through the `onPressed` when clicking on the app bar items, but not by scrolling with the mouse. Probably by adding a listener to figure out where we are and dynamically update the title should solve it, but might be out of scope and I'm fine with that as well :)

Reply:
This feature is now added.

https://user-images.githubusercontent.com/13782003/158882922-481f4780-923b-46bc-b10b-7dc0cec735fb.mov